pkg/cmd/pr/create/create_test.go

```diff
@@ -300,6 +306,21 @@ func Test_createRun(t *testing.T) {
 			},
 			expectedOut: "https://github.com/OWNER/REPO/pull/12\n",
 		},
+		{
+			name: "dry-run",
+			tty:  true,
+			setup: func(opts *CreateOptions, t *testing.T) func() {
+				opts.TitleProvided = true
+				opts.BodyProvided = true
+				opts.Title = "my title"
+				opts.Body = "my body"
+				opts.HeadBranch = "feature"
+				opts.DryRun = true
+				return func() {}
+			},
+			expectedOut:    "Would have created a Pull Request with map[baseRefName:master body:my body draft:false headRefName:feature maintainerCanModify:false title:my title]",
```

Comment:
I don't really like dropping Golang serialised output into our UX. I think we should go back to the issue and discuss what this output should look like. I think this needs some more care, for example what will the output look like with an extremely long text body.